README.md

```diff
@@ -87,97 +93,141 @@ However, this is always optional, and usually not necessary.
 
 ---Creates a new note and uses the last visual selection as the title while replacing the selection with a link to the new note
 --
+---@param options table additional options
 ---@param path? string path to explicitly specify the notebook
+require("zk.commands").new_from_title_selection(options, path)
+
+---Creates a new note and uses the last visual selection as the content while replacing the selection with a link to the new note
+--
 ---@param options table additional options
-require("zk").new_link(path, options)
+---@param path? string path to explicitly specify the notebook
+require("zk.commands").new_from_content_selection(options, path)
+
+
+---cd into the notebook root
+require("zk.commands").cd(path)
 
 ---Opens a Telescope picker
 --
+---@param options table additional options
 ---@param path? string path to explicitly specify the notebook
+---@see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zklist
+require("zk.commands").notes(options, path)
+
+---Opens a Telescope picker
+--
 ---@param options table additional options
+---@param path? string path to explicitly specify the notebook
 ---@see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zklist
-require("zk").list(path, options)
+require("zk.commands").backlinks(options, path)
 
 ---Opens a Telescope picker
 --
+---@param options table additional options
+---@param path? string path to explicitly specify the notebook
+---@see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zklist
+require("zk.commands").links(options, path)
+
+---Opens a Telescope picker, filters for notes that match the text in the last visual selection
+--
+---@param options table additional options
 ---@param path? string path to explicitly specify the notebook
+---@see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zklist
+require("zk.commands").match(options, path)
+
+---Opens a Telescope picker
+--
 ---@param options table additional options
+---@param path? string path to explicitly specify the notebook
 ---@see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zktaglist
-require("zk").tag.list(path, options)
+require("zk.commands").tags(options, path)
 ```
 
 *Examples:*
 ```lua
-require("zk").new(nil, { dir = "daily" })
-require("zk").list(nil, { createdAfter = "3 days ago", tags = { "work" } })
-require("zk").new_link() -- this will use your last visual mode selection
+require("zk.commands").new({ dir = "daily" })
+require("zk.commands").notes({ createdAfter = "3 days ago", tags = { "work" } })
+require("zk.commands").new_from_title_selection() -- this will use your last visual mode selection
 ```
 
 As you can see, the `path` is optional, and can usually be omitted; see [Notebook Directory Discovery](#notebook-directory-discovery).
 
 ### Telescope
+It's not really necessary to use this interface, instead we recommend you to use the provided [commands](#commands).
```

Comment:
Yeah, at this point we could probably just remove it from the documentation to avoid confusion. There aren't really any pros in using the telescope API directly.